releasenotes/notes/logs-fix-container-log-rotation-0f15727384e0048a.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Properly handle docker container logs in multiline mode
```

Comment:
This message is not specific enough IMO, here it seems like there were big issues on handling multiline logs with docker while there is an issue only at restarts (I think ?). Can we provide more details please ?